src/components/form/Admin.jsx

```diff
@@ -7,6 +7,7 @@ import { useSession } from "next-auth/react";
 import { api } from "@/utils/api";
 import toaster from "@/utils/toaster";
 import { STATUSES } from "@/data/Statuses";
+import { adminSchema } from "@/schemas/admin.ts";
```

Comment:
dont put the ts in the import should just be `@/schemas/admin`

call it just admin or schema cause we already know its from the admin schema file